r-package/grf/tests/testthat/test_regression_forest.R

```diff
@@ -2,6 +2,55 @@ library(grf)
 
 set.seed(1234)
 
+rm(list=ls())
+
+extract_samples <- function(tree) {
+  
+  # Keep only leaf nodes
+  leaf_nodes <- Filter(f = function(x) x$is_leaf, tree$nodes)
+  
+  # Leaf nodes' 'samples' are estimation samples
+  estimation_sample <- unlist(Map(f=function(x) x$samples, leaf_nodes)) - 1
+  
+  # Split = Drawn - Samples
+  split_sample <- base::setdiff(tree$drawn_samples, estimation_sample)
+  
+  # OOB samples
+  oob_sample <- base::setdiff(seq(2*length(tree$drawn_samples)-1), tree$drawn_samples)
```

Comment:
I think we can remove the `drawn_sample` and `oob_sample` keys, as they're not used in tests.